docs/faq.md

```diff
@@ -313,21 +313,38 @@ typically do not contribute to the meaning of a document and should therefore be
 
-One possible solution to this is to use [VS Code Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers), which allows you to setup a Linux-based environment. To run BERTopic effectively you need to be aware of two things:
+One possible solution is to use [VS Code Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers), which allow you to setup a Linux-based environment. To run BERTopic effectively you need to be aware of two things:
 
-- Make sure to use a Docker image specifically compiled for ARM64
-- Make sure to use `volume` instead of `mount-bind`, since the latter significantly reduces I/O speeds to disk
+1. Make sure to use a Docker image specifically built for arm64
+2. Make sure to use a *volume* instead of a *bind-mount*  
+   ℹ️ the latter significantly reduces disk I/O
 
-Using the pre-configured [Data Science Devcontainers](https://github.com/b-data/data-science-devcontainers) makes sure these setting are optimized. To start using them, do the following:
+Using the pre-configured [Data Science Dev Containers](https://github.com/b-data/data-science-devcontainers) makes sure these setting are optimized. To start using them, do the following:
 
-- Install and run Docker
-- Install `python-base` or `python-scipy` [devcontainer](https://github.com/b-data/data-science-devcontainers)
-- ℹ️ Change PYTHON_VERSION to 3.11 in the `devcontainer.json` to work with the latest version of Python 3.11 (currently 3.11.8)
-- Open VS Code, build the container and start working
-- Note that data is persisted in the container
-    - When using an unmodified devcontainer.json: work in `/home/vscode` which is the `home` directory of user `vscode`
-    - Python packages are installed to the home directory by default. This is due to env variable `PIP_USER=1`
-    - Note that the directory `/workspaces` is also persisted
+* Install and run Docker
+* Clone repository [data-science-devcontainers](https://github.com/b-data/data-science-devcontainers)
+* Open VS Code, build the `Python base` or `Python scipy` container and start working  
+  ℹ️ Change `PYTHON_VERSION` to `3.11` in the respective `devcontainer.json` to work with the latest patch release of Python 3.11
+* Note that data is persisted in the container
+  * When using an unmodified `devcontainer.json`: Work in `/home/vscode`  
+    👉 This is the *home directory* of user `vscode`
+  * Python packages are installed to the home directory by default  
+    👉 This is due to env variable `PIP_USER=1`
+  * Note that the directory `/workspaces` is also persisted
 
+## **Speaking of Dev Containers: Do they support GPU acceleration?**
```

Comment:
Could you make this title a bit more specific and not dependent on the content of the previous question? The users will likely see a list of questions and the title "Speaking of Dev Containers" is dependent on the content of the previous title and not the title itself. 

Instead, you could also make it a subsection instead to showcase that it is indeed related to the previous section.

Reply:
By subsection you mean

```
### **Speaking of Dev Containers: Do they support GPU acceleration?**
```

Correct?